### ISSUE TITLE
Possible error in "Learn CSS" course chapter 9 regarding unpacking of "flex: 1"

### ISSUE THREAD

**Issue by remigius42** [GoogleChrome/web.dev#5537]
**Describe the bug**
Thank you for providing the "Learn CSS" course!
While reading [chapter 9 about Flexbox](https://web.dev/learn/css/flexbox/), I came across the [example about the unpacking of `flex: 1`](https://web.dev/learn/css/flexbox/#controlling-space-inside-flex-items), which might be incorrect.
The course material states

> To force all of the items to be a consistent size and ignore the size of the content change flex:auto to flex: 1. in the demo.
>
> This unpacks to:
>
>  * flex-grow: 1: items can grow larger than their flex-basis.
>  * flex-shrink: 0: items can't shrink smaller than their flex-basis.
>  * flex-basis: 0: items have a base size of 0.

which by my understanding of the [spec](https://drafts.csswg.org/css-flexbox-1/#flex-common) should be  

* `flex-shrink: 1`: items can shrink smaller than their flex-basis`. 

(see also the [MDN documentation regarding the `flex` property](https://developer.mozilla.org/en-US/docs/Web/CSS/flex#syntax)).

**To Reproduce**
1. Go to https://drafts.csswg.org/css-flexbox-1/#flex-common
2. Compare it to the spec at https://drafts.csswg.org/css-flexbox-1/#flex-common

**Expected behavior**

* The second list item should probably be changed to  
    `flex-shrink: 1`: items can shrink smaller than their flex-basis` 
* The sentence before the list seems incorrect and could be changed to "In the demo this unpacks to:"

**Comment by remigius42** [comment]
I created a [suggestion for a possible fix](https://github.com/remigius42-forks/web.dev/commit/1834c313bf675555da4b74c1e0acf7d4ae25a1e5) and can open a pull request if this helps.

**Comment by rachelandrew** [comment]
@remigius42 I think you are right - please open a PR and I'll take a look at it :)

**Comment by remigius42** [comment]
Dear @rachelandrew, thank you for your quick response. I opened pull request https://github.com/GoogleChrome/web.dev/pull/5539 based on my [commit mentioned above](https://github.com/remigius42-forks/web.dev/commit/1834c313bf675555da4b74c1e0acf7d4ae25a1e5). However, I should probably add, that I guestimated your commit conventions based on the most recent commits I looked at and therefore my commit might have to be rewritten.
